types/prettier/index.d.ts

```diff
@@ -92,7 +92,7 @@ export interface RequiredOptions extends doc.printer.Options {
     /**
      * Specify which parser to use.
      */
-    parser: BuiltInParserName | CustomParser;
+    parser: BuiltInParserName | CustomParserName | CustomParser;
```

Comment:
This could possibly break auto completion for `BuiltInParserName` :thinking:  
Maybe we can work around this with this _hack_ https://github.com/microsoft/TypeScript/issues/29729#issuecomment-471566609

Reply:
Updated!